soh/soh.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is the xcworkspace something that gets checked in generally? i don't think we are using workspaces here, so might just be best to ignore `project.xcworkspace` entirely?